app/src/main/java/sw19/moring03/paint/tools/CircleTool.java

```diff
@@ -18,11 +19,12 @@ public CircleTool() {
         radius = 0;
     }
 
-    public CircleTool(int col, int strkW) {
+    public CircleTool(int col, int strkW, PathEffect effect) {
```

Comment:
Can you please set a default pathEffect in the default constructor? Just to be sure that all fields are initialized and we don't run into any NullPointerExceptions later ^^